src/script/auth/component/EntropyCanvas.tsx

```diff
@@ -19,89 +19,90 @@
 
 import {CSSObject} from '@emotion/react';
 import {MouseEvent, useRef, useEffect, useState} from 'react';
-import {flushSync} from 'react-dom';
 import {usePausableInterval} from '../../hooks/usePausableInterval';
+import {EntropyData, EntropyFrame} from '../../util/Entropy';
 
 interface CanvasProps {
   css?: CSSObject;
   'data-uie-name'?: string;
-  onProgress: (entropyData: [number, number][], percent: number, pause: boolean) => void;
-  onSetEntropy: (entropyData: [number, number][]) => void;
+  onProgress: (entropyData: EntropyData, percent: number, pause: boolean) => void;
   sizeX: number;
   sizeY: number;
+  // minimum number of frames
+  min_frames: number;
+  // minimum bits of overall estimated entropy
+  min_entropy_bits: number;
 }
 
 const EntropyCanvas = (props: CanvasProps) => {
-  const {sizeX, sizeY, onSetEntropy, onProgress, css, ...rest} = props;
-  const canvasRef = useRef(null);
+  const {sizeX, sizeY, onProgress, css, min_entropy_bits, min_frames, ...rest} = props;
```

Comment:
can we change these to camelCase
```suggestion
  const {sizeX, sizeY, onProgress, css, minEntropyBits, minFrames, ...rest} = props;
```